React/CoreModules/RCTDevSettings.mm

```diff
@@ -225,6 +225,27 @@ - (void)initialize
   }
 #endif
 
+#if RCT_DEV_MENU
```

Comment:
Should this move in under the `RCT_DEV_SETTINGS_ENABLE_PACKAGER_CONNECTION` macro like the reload command?

Reply:
Yes, good catch. Fixed ✅